README.md

```diff
@@ -70,6 +70,9 @@ By calling `AddWhatsappTemplateMessage` and providing `Template`, sender name, r
 
 See Examples folder for more examples.
 
+## Sending Interactive Whatsapp message
+See the examples folder for an example.
```

Comment:
(arguebaly you an add a really simple example here or a direct link to the folder but thats just me)

Reply:
Added a direct link to the file.